src/args.rs

```diff
@@ -104,13 +103,6 @@ pub enum Action {
     /// Run continuously, watching the repository for changes and deploying as soon as they
     /// happen. Can be ran with `--dry-run`
     Watch,
-
-    /// Generate shell completions
-    GenCompletions {
-        /// Set the shell for generating completions [values: bash, elvish, fish, powerShell, zsh]
-        #[clap(long, short)]
-        shell: Shell,
-    },
```

Comment:
I think we can keep this - not much reason to break existing setups.

Reply:
Software should keep simple and stupid. If we have a better way to distribute shell completions, reducing the ways is right even it will cause broken changes.

For Rust code, removing runtime completions generation means cutting out one dependency.

Reply:
Aiming to reduce binary size is a fair argument, but I think backward compatibility is more important. I'm sure users involved in #65 and #107 won't be thrilled about having to change their configuration.

Reply:
Considering that not everyone are using a package manager to install dotter, I agree to keep the runtime generation.

However, **convention over configuration**. Placing completion scripts to directories in convention is better than writing generation commands in configuration. I wish users would do that.